<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip { color: #fff; background-color: black; border: none; }

#InfoDialog
{
background: white;
border: 2px solid rgb(100, 100, 100);
}

#wHeader
{
margin-left: 1px;
margin-right: 1px;
border: 0px;
border-bottom: 2px solid rgb(241, 240, 240);
}

#wRecent1
{
margin-left: 1px;
margin-right: 1px;
border: 0px;
border-bottom: 2px solid rgb(241, 240, 240);
}

#wRecent2
{
margin-left: 1px;
margin-right: 1px;
border: 0px;
border-bottom: 2px solid rgb(241, 240, 240);
}

#wRecent3
{
margin-left: 1px;
margin-right: 1px;
border: 0px;
border-bottom: 2px solid rgb(241, 240, 240);
}

#wUsage
{
margin-left: 1px;
margin-right: 1px;
border: 0px;
border-bottom: 2px solid rgb(217, 217, 217);
}

#lRecentlyUpdated
{
padding-left: 14px;
background: rgb(241, 240, 240);
color:  rgb(166, 166, 166);
border-left: 1px solid rgb(100, 100, 100);
border-right: 1px solid rgb(100, 100, 100);
}

#lSyncUpdated
{
color:  rgb(166, 166, 166);
}

#wBottom
{
background: rgb(242, 242, 242);
border-bottom: 2px solid rgb(100, 100, 100);
border-left: 1px solid rgb(100, 100, 100);
border-right: 1px solid rgb(100, 100, 100);
}

#sActiveTransfers
{
border: 0px;
margin-left: 1px;
margin-right: 1px;
padding: 0px;
}

#lQueued
{
color: rgb(102, 102, 102);
}

#lDownloads
{
color: rgb(102, 102, 102);
}

#lUploads
{
color: rgb(102, 102, 102);
}

#bUploads
{
color: rgb(102, 102, 102);
}

#bDownloads
{
color: rgb(102, 102, 102);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wHeader" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lHeader">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/images/preferences_mega_logo.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bPause">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
     width: 24px;
     height: 24px;
	 subcontrol-position:  center;
 }

  QCheckBox::indicator:checked
  {
    image: url(://images/tray_play_ico.png);
  }
  QCheckBox::indicator:unchecked
  {
    image: url(://images/tray_pause_ico.png);
  }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSettings">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin: 4px;
border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>://images/tray_preferences_ico.png</normaloff>://images/tray_preferences_ico.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="sActiveTransfers">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pUpdated">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>95</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>://images/tray_updated_large_ico.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lSyncUpdated">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MEGAsync is up to date</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pUpdating">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wTransfers" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>120</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="ActiveTransfer" name="wTransfer1" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ActiveTransfer" name="wTransfer2" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1">
          <item>
           <widget class="QPushButton" name="bDownloads">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin: 4px;
border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>://images/tray_download_ico.png</normaloff>://images/tray_download_ico.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lDownloads">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bUploads">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin: 4px;
border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>://images/tray_upload_ico.png</normaloff>://images/tray_upload_ico.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lUploads">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lRecentlyUpdated">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RECENTLY UPDATED</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="recentLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="RecentFile" name="wRecent1" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RecentFile" name="wRecent2" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RecentFile" name="wRecent3" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wUsage" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>17</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>17</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lTotalUsed">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Usage: Data temporarily unavailable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lPercentageUsed">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="UsageProgressBar" name="pUsage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wBottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="bSyncFolder">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string>Syncs</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>://images/tray_folder_ico.png</normaloff>://images/tray_folder_ico.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bOfficialWeb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string>MEGA website</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>://images/tray_web_ico.png</normaloff>://images/tray_web_ico.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActiveTransfer</class>
   <extends>QWidget</extends>
   <header>ActiveTransfer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecentFile</class>
   <extends>QWidget</extends>
   <header>RecentFile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UsageProgressBar</class>
   <extends>QWidget</extends>
   <header>UsageProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
